configuration/concepts/total_joints.csv

```diff
@@ -0,0 +1,26 @@
+Uuid,Void/Retire,Fully specified name:en,Short name:en,Fully specified name:es,Short name:es,Description:en,Data class,Data type,Answers,Members
+f93d0420-4771-4988-bf61-8b9461ec4005,,Shoulder,Shoulder,Hombro,Hombro,,Misc,N/A,,
+245d28b0-d1c5-4b85-a2fd-94cdd610130e,,Hip,Hip,Cadera,Cadera,,Misc,N/A,,
+66e3e1d1-3daf-4ac3-8d79-85d0755abf00,,Knee,Knee,Rodilla,Rodilla,,Misc,N/A,,
+2ed9c8c0-1534-4d2d-a0d8-0987a9607d34,,Joint,Joint,Articulación,Articulación,,Misc,Coded,Shoulder; Hip; Knee,
+a11a2309-3600-4ea8-957c-18591654ad37,,Both,Both,Ambos,Ambos,,Misc,N/A,,
+11cd4cc3-0d09-4771-aac3-40e0ab4342b5,,Body Side,Body Side,Lado del cuerpo,Lado del cuerpo,,Misc,Coded,Right; Left; Both,
+55ad06d0-f5d7-4bfc-9a3e-8a017047da26,,Patient's Preference,Patient's Preference,Preferencia del paciente,Preferencia del paciente,,Misc,Coded,Right; Left,
+31420631-a70e-4658-9aeb-fdac4a4f83dc,,Duration of Pain,Duration of Pain,Duración del dolor,Duración del dolor,,ConvSet,N/A,,Hour(s); Weeks; Minutes; Months
+935c40a8-b737-44c1-b3a4-13ed2776c09d,,Normal,Normal,Normal,Normal,,Misc,N/A,,
```

Comment:
Is this not a duplicate [here](https://github.com/globalhealthcoalition/openmrs-config-ghc/blob/6ea3a8886f4704bb0182da9658fc02021d96e5ca/configuration/concepts/lab_tests.csv#L4)

Reply:
It was